BC_buymenu/functions/fn_addBoatInteraction.sqf

```diff
@@ -1,16 +1,16 @@
 params ["_object"];
 
 // boat carrying
-_carryAssaultBoat = ["CarryBoatAction", "Carry Boat", "",
+private _carryAssaultBoat = ["CarryBoatAction", "Carry Boat", "",
   {[_target] spawn GRAD_carryBoat_fnc_pickupBoat;},
```

Comment:
If you are going through every file anyways, convert to 4 spaces indentation.

Reply:
Sry not going to fix indentations.